rs/state_machine_tests/src/lib.rs

```diff
@@ -1345,10 +1345,7 @@ impl StateMachine {
                 }
             })
             .collect();
-        self.canister_http_pool
-            .write()
-            .unwrap()
-            .apply_changes(changeset);
+        self.canister_http_pool.write().unwrap().apply(changeset);
```

Comment:
May be too much to ask, but should variables and arguments (such as `changeset`) also be renamed?

Reply:
yeah, this will be to much in this MR. It will organically happen in follow-up MRs.